src/inverseRobot.js

```diff
@@ -7,7 +7,16 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
-}
+  const newRobot = {};
 
+  for (const key in robot) {
+    if (robot[key] in newRobot) {
+      return null;
+    } else {
+      newRobot[robot[key]] = key;
+    }
```

Comment:
```suggestion
    if (robot[key] in newRobot) {
      return null;
    }
    
    newRobot[robot[key]] = key;
```
there's no need in else